app/views/think_feel_do_dashboard/arms/show.html.erb

```diff
@@ -19,13 +19,17 @@
           </li>
         <% end %>
         <li>
-          <% if can? :index, ContentModules::LessonModule %>
+          <% if @lesson && can?(:index, ContentModules::LessonModule) %>
             <%= link_to "Lesson Modules", (defined?(think_feel_do_engine) ? think_feel_do_engine.arm_lessons_path(@arm) : root_path) %>
+          <% elsif can? :index, ContentModules::LessonModule %>
+            <%= link_to "Lesson Modules*", "javascript:void(0);", data: { confirm: "A learn tool has to be created in order to access this page." } %>
```

Comment:
it has the link do nothing.  I could use just use `javascript:;`, but either seems to work.  We could use `#` but that then that augments the url and scrolls the page.  I like using `javascript:void(0)`